NosTests/Service/NostrEntity/NostrEntityTests.swift

```diff
@@ -0,0 +1,171 @@
+import XCTest
+
+class NostrEntityTests: XCTestCase {
+    /// Example taken from [NIP-19](https://github.com/nostr-protocol/nips/blob/master/19.md)
+    @MainActor func test_npub() throws {
```

Comment:
I don't think these need to be marked @MainActor if they aren't async?

Reply:
Copy/paste error. I'll fix this.

Reply:
✅